### PR TITLE
Springboot example update

### PR DESCRIPTION
**Pull request by sumitjoshi1989**

### CODE REVIEW

File: examples/springboot/src/main/java/io/airbrake/weather/WeatherController.java [hunk 1]
```diff
@@ -15,31 +13,31 @@
...
+        return weatherService.getWeather(location);
       
     }
 
     @GetMapping("/weather1/{location}")
-    public String weather1(@PathVariable String location) {
+    public String weatherTryCatch(@PathVariable String location) {
     
-        return svc.GetWeather1(location);
+        return weatherService.getWeatherTC(location);
```
Comment (mmcdaris) [inline examples/springboot/src/main/java/io/airbrake/weather/WeatherController.java:40]: Please update this function name to getWeatherTryCatch

**Review by mmcdaris — COMMENTED** [review]
Nice work. One suggested edit. Can you please confirm the new version of the example app still runs when provided a real project id and key?

File: examples/springboot/src/main/java/io/airbrake/weather/WeatherController.java (hunk 1 above)
Comment (sumitjoshi1989) [reply examples/springboot/src/main/java/io/airbrake/weather/WeatherController.java]: Changed the function name. Now its working fine

**Review by mmcdaris — APPROVED** [review]
Nice update @sumitjoshi1989 👍

**Pull request merged**